minifi-docker/dockerhub/Dockerfile

```diff
@@ -40,7 +40,8 @@ ADD sh/ ${MINIFI_BASE_DIR}/scripts/
 RUN curl -fSL $MINIFI_BINARY_URL -o $MINIFI_BASE_DIR/minifi-$MINIFI_VERSION-bin.tar.gz \
 	&& echo "$(curl $MINIFI_BINARY_URL.sha256) *$MINIFI_BASE_DIR/minifi-$MINIFI_VERSION-bin.tar.gz" | sha256sum -c - \
 	&& tar -xvzf $MINIFI_BASE_DIR/minifi-$MINIFI_VERSION-bin.tar.gz -C $MINIFI_BASE_DIR \
-	&& rm $MINIFI_BASE_DIR/minifi-$MINIFI_VERSION-bin.tar.gz
+	&& rm $MINIFI_BASE_DIR/minifi-$MINIFI_VERSION-bin.tar.gz \
+	&& ln -s $MINIFI_BASE_DIR/minifi-$MINIFI_VERSION $MINIFI_HOME
 
 RUN chown -R minifi:minifi $MINIFI_HOME
```

Comment:
The container no longer successfully starts as before with the symbolic link and it appears the container cannot run with the following:
> /bin/sh: /opt/minifi/minifi-current/bin/minifi.sh: Permission denied

Believe the chown will also need to be modified to traverse those links with a -L, I think.

Reply:
Apologies! The -L flag did fix this. I'll submit